circle.yml

```diff
@@ -15,10 +15,11 @@ dependencies:
     - "~/.cabal"
 
   override:
+    - rm -rf /home/ubuntu/.go_workspace/src
     - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
     - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
```

Comment:
The location of the project in the `$GOPATH` should match its import path, otherwise you'll probably run into weird behavior.

I usually do something like:
```yaml
environment:
  GO_PROJECT: "$HOME/.go_workspace/src/github.com/Originate/<repository>"

dependencies:
    override:
      # Symlink the project into the $GOPATH
      - mkdir -p "$(dirname "$GO_PROJECT")"
      - if [[ ! -e "$GO_PROJECT" ]]; then ln -s $HOME/6fusion-online-go "$GO_PROJECT"; fi

```